File: app/src/main/java/android/pdf4664/notepad/activities/WebActivity.java
package android.pdf4664.notepad.activities;

import android.content.Intent;
import android.pdf4664.notepad.R;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.webkit.WebView;

public class WebActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_web);

        Intent intent = getIntent();
        String content = intent.getExtras().getString("file");

        WebView web = (WebView)findViewById(R.id.web_view);
        web.loadData(content, "text/html; charset=utf-8", "utf-8");
    }
}

File: app/src/main/java/org/pdf4664/notepad/activities/WebActivity.kt
package org.pdf4664.notepad.activities

import org.pdf4664.notepad.R
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.webkit.WebView

class WebActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_web)

        val intent = intent
        val content = intent.extras!!.getString("file")

        val web = findViewById<View>(R.id.web_view) as WebView
        web.loadData(content, "text/html; charset=utf-8", "utf-8")
    }
}
